integration-tests/test-suite/partials/scroll.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    AET
+
+    Copyright (C) 2013 Cognifide Limited
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<suite company="aet" domain="http://192.168.123.100:9090/sample-site/sanity/" name="main"
+  project="aet">
+  <!-- Scroll-Modifier BEGIN -->
+  <test name="S-modifier-scroll-default-position-bottom">
+    <collect>
+      <open/>
+      <sleep duration="1500"/>
+      <scroll/>
```

Comment:
What is the size of taken screenshots? If you're not using resolution modifier I think that the results may be inconsistent across different tests runs - please add `resolution` tag, I guess you should use some low `height` value to test if scrolling works properly :)

Reply:
Done!